test.js

```diff
@@ -43,7 +43,7 @@ test('listDirectories()', async t => {
 		await listDirectories([0, 1]);
 		fail();
 	} catch ({name}) {
-		t.equal(name, 'TypeError', 'should fail when it takes a non-string argument.');
+		t.equal(name.indexOf('TypeError'), 0, 'should fail when it takes a non-string argument.');
```

Comment:
Let's check [`error.code`](https://nodejs.org/docs/latest-v10.x/api/errors.html#errors_error_code_1) instead.

Reply:
Done - the PR has been updated.